`primero` Add `metadata` in `getCases` state

mtuchi opened the pull request:
### Description 
Added `metadata` in `getCases`  state

taylordowns2000 reviewed (changes requested):
Please adjust the changeset here as we've just added `metadata` to _every single function_ in the adaptor.

mtuchi commented:
> Please adjust the changeset here as we've just added `metadata` to _every single function_ in the adaptor.

Addressed ➡️  https://github.com/OpenFn/adaptors/pull/84/commits/33742bf4049de77076a67a0c42198d82a7e0acce
